src/System/Console/GetOpt/Generics/Internal.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE ConstraintKinds     #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Is this actually necessary? There are a few places that use `forall`, but they don't seem to be doing anything with scoping.